packages/web/src/scss/tools/_dictionaries.scss

```diff
@@ -193,7 +193,7 @@
             // @deprecated CSS modifier classes are deprecated and will be removed in the next major version
```

Comment:
Just wonder, this will be deprecated by what ticket?

Reply:
DS-1183